setup.cfg

```diff
@@ -24,9 +22,7 @@ classifier =
 [options]
 install_requires =
     six>=1.9.0
```

Comment:
I'd appreciate if the requirement on `six` would get dropped together with Python 2.7 support as well.

Reply:
Yeah, it could be nice. Totally agree but I'd prefer to make it in a separate PR to keep this one focused. After removing Python 2 dependency we could have a bunch of other different improvements. Created an issue to not forget about it: https://github.com/jd/tenacity/issues/305